src/server/dfly_main.cc

```diff
@@ -438,6 +439,114 @@ bool ShouldUseEpollAPI(const base::sys::KernelVersion& kver) {
   return true;
 }
 
+string get_cgroup_path(void) {
+  // Begin by reading /proc/self/cgroup
+
+  auto cg = io::ReadFileToString("/proc/self/cgroup");
+  CHECK(cg.has_value());
+
+  // Here, we assume that CGroup v1 is being used. This
+  // is quite likely, as CGroup v1 was introduced back in 2015.
+
+  // strip 0::<path> into just <path>, and newline.
+  auto cgv = std::move(cg.value());
+  return cgv.substr(3, cgv.length() - 3 - 1);
```

Comment:
I would not want to assume that chgroup starts with zero.
it could also have arbitrary number: https://manpages.ubuntu.com/manpages/bionic/en/man7/cgroup_namespaces.7.html
Please find the second ":" and use it to fetch the path.

Reply:
Interesting. Here it says that cgroups should always start with `0::`

https://docs.kernel.org/admin-guide/cgroup-v2.html#processes

Which one will it be? :thinking: 

BTW, I ran their example and also got `0::`.